broker/config/database.js

```diff
@@ -1,18 +1,17 @@
 // Read private configuration settings from .env file into process.env
 //
-const dotenv = require("dotenv");
-dotenv.config();
+require("dotenv").config({ path: `${__dirname}/../../.env` });
 
 // Export configuration settings with some values set to defaults.
 // NOTE: login/password settings should not be defaulted.
 //
 module.exports = {
     // For Sequelize (defaults below for PostgreSQL)
     //
-    dbName: process.env.DB_NAME || 'massvaxx',
-    dialect: process.env.DB_DIALECT || 'postgres',
-    host: process.env.DB_HOST || 'localhost',
-    password: process.env.DB_PASSWORD,
-    port: process.env.DB_PORT || 5432,
-    username: process.env.DB_USERNAME
+    dbName: process.env.BROKER_DB_NAME || 'massvaxx',
```

Comment:
Should we move .env_template from /broker to / and update it to indicate the required settings in the root .env?  I think we should make sure folks have clear documentation of what settings need to be in .env to build